main.py

```diff
@@ -36,10 +36,16 @@ class Determinant(NamedTuple):
     beta: Spin_determinant
 
 class DetDiff(NamedTuple):
+    """
+    object to hold holes and particles of both spins between two determinants
+    """
 
     holes: Determinant
     particles: Determinant
     def __repr__(self):
+        """
+        transpose of default repr is easier to read quickly
+        """
         return(f'DetDiff(hA={self.holes.alpha}, pA={self.particles.alpha}; hB={self.holes.beta}, pB={self.particles.beta})')
```

Comment:
It seems strange to me that the `__rper__` return a `DetDiff`. Also `detDiff` doesn't have an `hA` argument in the constructor. How can this work?
Also, I'm not sure about this `;`. 

If it is not used in this file, for now, I vote for keeping it only in your fork. Would like to keep this branch as minimal as possible

Reply:
not used, so I removed it